lib/config.rb

```diff
@@ -0,0 +1,6 @@
+# constants
+DATE_FMT = "%Y-%m-%d".freeze                                    # ISO
```

Comment:
Can you move those constants under `GnuCash` module? And place them inside of `lib/gnucash.rb`?

Reply:
I'm fine if you will move it to `lib/gnucash/constants.rb` though or `lib/gnucash/core.rb` for example.